ArduCopter/Copter.cpp

```diff
@@ -413,6 +413,28 @@ bool Copter::set_desired_speed(float speed)
 }
 
 #if MODE_AUTO_ENABLED == ENABLED
+// add an additional offset to vehicle's target position
+// units are m or m/s in NED frame
+bool Copter::set_auto_pos_offset(const Vector3f &pos_offset_NED)
+{
+    if (flightmode != &mode_auto) {
+        return false;
+    }
+    wp_nav->set_pos_offset(pos_offset_NED.topostype());
```

Comment:
I was thinking about this after our discussion. There isn't any reason we can't do this like guided where we use:
1. Pos, Vel, Accel
2. Vel, Accel

We just leave the later ones zero if we don't what the increased reaction time they provide.

Reply:
we've changed the interface now to allow callers to provide pos, vel and accel target offsets